lighthouse-core/environment/cli.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+module.exports = {
+  name: 'CLI',
+};
```

Comment:
I was also considering having one `const ENVIRONMENT = 'replace-me'` in the `runner.js` instead of three `environment/` files, but replacing imports instead of values seemed a tiny bit cleaner.

Reply:
yeah I agree abstractly, unfortunately it doesn't seem as easy as I expected in browserify 😆 

is there any simpler way with less manual? I'm no browserify expert, maybe something similar to the webpack define plugin or simple shim replacement without custom transform?

with the audits/gatherers we manually do something fancy without a transform which is what makes me hopeful :D

Reply:
> webpack define plugin

that was a good tip, thanks! (after a lot of googling) I found similar solution for browserify 👌